### PR TITLE
Translation (Combine strings and variables) - jump_to_xy block

### PR DESCRIPTION
**Pull request by nkiruka**
Each of the following strings below will be combined with variables to give translators the option to translate the entire string:

Implementation for the jump_to_xy block (- jump to 0 over 0 down )
Screen Shot 2019-08-26 at 11 56 42 AM

The jump_to_xy block was updated to include the newly modified string.

To ensure the new string is parsed correctly, the interporlateMsg function in blocks.js(blockly) was modified to include the option of adding a field dropdown and/or a text input.

[jira](https://codedotorg.atlassian.net/secure/RapidBoard.jspa?rapidView=28&modal=detail&selectedIssue=FND-119&assignee=5c5087d098c1ac41b43286f4)

- [PR 30582](https://github.com/code-dot-org/code-dot-org/pull/30582) move forward by 100 pixels (the first six are all in Actions) 
- [PR 30584](https://github.com/code-dot-org/code-dot-org/pull/30584) right by 90 degrees 
- [PR 30584](https://github.com/code-dot-org/code-dot-org/pull/30584) left by 90 degrees 
- [PR 30484](https://github.com/code-dot-org/code-dot-org/pull/30484) forward by 100 pixels 
- [PR 30460](https://github.com/code-dot-org/code-dot-org/pull/30460) to the top left position

### CODE REVIEW

File: apps/i18n/turtle/en_us.json [hunk 1]
```diff
@@ -43,8 +43,11 @@
   "hideTurtle": "hide artist",
   "jump": "jump",
   "jumpTo": "jump to",
+  "jumpToLeftPosition": "jump to the top left position",
```
Comment (bethanyaconnor) [inline apps/i18n/turtle/en_us.json]: Don't we need to change the code to use these new strings?

File: apps/i18n/turtle/en_us.json [hunk 2]
```diff
@@ -43,8 +43,11 @@
...
+  "jumpToOverDown": "jump to {x} over {x} down",
```
Comment (Hamms) [inline apps/i18n/turtle/en_us.json]:
```suggestion
  "jumpToOverDown": "jump to {x} over {y} down",
```

File: apps/i18n/turtle/en_us.json (hunk 1 above)
Comment (Hamms) [reply apps/i18n/turtle/en_us.json]: I expect we also need to change this to be something more like `"jump to the {pos} position"`

**Comment by joshlory** [comment]
I don't think we want to check in the `package-lock.json` diff? This repo already uses `yarn.lock`.

File: apps/i18n/turtle/en_us.json [hunk 3]
```diff
@@ -43,6 +43,7 @@
   "hideTurtle": "hide artist",
   "jump": "jump",
   "jumpTo": "jump to",
```
Comment (Hamms) [inline apps/i18n/turtle/en_us.json]: can this be removed?

**Review by Hamms — APPROVED** [review]
LGTM, modulo one question

File: apps/i18n/turtle/en_us.json (hunk 3 above)
Comment (nkiruka) [reply apps/i18n/turtle/en_us.json]: yep!  In the process of pushing commit :-)

**Review by Hamms — APPROVED** [review]

**Review by joshlory — APPROVED** [review]
LGTM!

**Pull request merged**